infrastructure/aws/staging/ecs-task-definition-web.json

```diff
@@ -19,6 +19,10 @@
             ],
             "essential": true,
             "environment": [
+                {
+                    "name": "COOKIE_AUTH_ENABLED",
+                    "value": "1"
+                },
```

Comment:
Do we also need to remove this before merge @khvn26 ?

Reply:
Yes.